buildmachine/install-load-tests.ps1

```diff
@@ -0,0 +1,100 @@
+#Requires -RunAsAdministrator
+
+[CmdletBinding()]
+param (
+    [Parameter()][string]$JMeterVersion = "5.4.1",
+    [Parameter()][string]$DownloadFolder = "C:\download",
+    [Parameter()][string]$JMeterPath = "C:\tools\jmeter",
+    [Parameter()][string]$PluginsList = "jpgc-graphs-basic,jpgc-casutg,jpgc-prmctl"
+)
+
+function ExtractArchive([string]$archiveFile, [string]$outputFolder) {
+    Write-Host "    Extract archive '$archiveFile' to '$outputFolder'" -ForegroundColor Blue
+
+    $extractCommand = "7z x $archiveFile -o$jmeterSource -r -aoa"
```

Comment:
JMeter sa dá stiahnuť aj vo [formáte ZIP](https://downloads.apache.org/jmeter/binaries/). Takže asi by bolo lepšie zobrať ten a zjednoduší sa rozbaľovanie. Nemusíš to robiť na dvakrát a nepotrebuješ ani 7-Zip, pretože to spravíš čisto v Powershelli funkciou [Expand-Archive](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/expand-archive?view=powershell-7.1).

Reply:
Kua s tým `tgz` som sa drbal asi hodinu. Jak to že som prehliadol, že majú aj `zip`? Nevedel si mi to povedať skôr? :-D

Reply:
Mea culpa, mea maxima culpa. 🤣